homeassistant/components/ecovacs/config_flow.py

```diff
@@ -77,11 +94,21 @@ async def async_step_user(
                                 type=selector.TextSelectorType.PASSWORD
                             )
                         ),
-                        vol.Required(CONF_COUNTRY): vol.All(vol.Lower, cv.string),
-                        vol.Required(CONF_CONTINENT): vol.All(vol.Lower, cv.string),
+                        vol.Required(CONF_COUNTRY): selector.CountrySelector(),
+                        vol.Optional(CONF_CONTINENT): selector.SelectSelector(
+                            selector.SelectSelectorConfig(
+                                options=list(set(COUNTRIES_TO_CONTINENTS.values())),
```

Comment:
Should we translate them?
Currently, we have eu,na,as,ww
But I don't know if there is one missing and Ecovacs is not sharing any information, therefore custom values are enabled